### DISCUSSION TITLE
Could I use multiple package manager?

### DISCUSSION THREAD

**Discussion by kyun** [vercel/turborepo#3163] [Help] [answered]
Currently, I have 2 apps in my Turborepo. Their package manager is `yarn 1`.
Now I am going to add an app for AWS lambda. I want to use `yarn berry`  for new app only.

1. Is it good to manage AWS Lambda and Web Frontend code as monorepo?
2. Could I seperate package manager by app?

**Comment by mehulkar** [comment, marked as answer]
This is not possible at the moment, because Turborepo detects the package manager based on what's available in the root of the monorepo. Could you open an issue for this feature if it is still useful for you?

**Comment by sgammon** [comment]
```
mv bun.lockb bun.inert && turbo build ; mv bun.inert bun.lockb
```
